### ISSUE TITLE
Another coin, another block got lost

### ISSUE THREAD

**Issue by Hoellschrei** [DirtyHarryDev/miningcore#7]
Another try with a non hybrid coin (Dimecoin).

[2021-02-02 13:54:26.9567] [I] [dime] Submitting block 4320824 [5ee1cb9ea78365e18f8765f94bc98cb20dbc6180d823a8c6c6f8974c2b00494b]
[2021-02-02 13:54:27.0916] [D] [dime] ExecuteBatchAnyAsync("submitblock", "getblock")
[2021-02-02 13:54:27.1025] [D] [dime] UpdateJob()
[2021-02-02 13:54:27.1025] [D] [dime] GetBlockTemplateAsync()
[2021-02-02 13:54:27.1025] [D] [dime] ExecuteCmdAnyAsync("getblocktemplate")

[2021-02-02 13:54:28.0991] [W] [dime] Block 4320824 submission failed for pool dime because block was not found after submission

No block ever written to database. 
Is this fork working?

**Comment by DirtyHarryDev** [comment]
Yes is working, your block reverse hashing is wrong in coins json
